toptal/picasso
# feat(UserBadge): add support for custom name rendering

[FX-692](https://toptal-core.atlassian.net/browse/FX-692)

### How to test
- Check the relevant example in Storybook - http://localhost:9001/?path=/story/components-folder--userbadge#custom-name-rendering
- Add the new prop to an existing `UserBadge` component and make sure the passed component renders correctly
`renderName={() => <Link href='#'>Jacqueline Roque</Link>}`

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![image](https://user-images.githubusercontent.com/12392666/72085059-56ad2280-330d-11ea-862c-99f9b9529c62.png) | ![image](https://user-images.githubusercontent.com/12392666/72085078-62004e00-330d-11ea-91d5-fbf8213a6035.png) |

### Review
